rmgweb/main/tools.py

```diff
@@ -49,8 +49,8 @@ def moleculeToURL(molecule):
     mol = molecule.copy(deep=True)
     mol.clearLabeledAtoms()
     adjlist = mol.toAdjacencyList(removeH=False)
-    url = urllib.quote(adjlist)
-    return url
+    # Return adjlist without quoting because reverse() will automatically quote everything properly
+    return adjlist
```

Comment:
Good point. I'll change the name of the method.

I can also make the same changes to `groupToURL`, but that method is actually never used, so it won't have any effect.